compiler/testData/codegen/bytecodeText/constructors/innerPrimaryDefaults.kt

```diff
@@ -0,0 +1,11 @@
+class A(val s: String) {
+    inner class B(val x: Int = 0)
+}
+
+// @A.class
+// 1 public <init>\(Ljava/lang/String;\)V
+
+// @A$B.class
+// 0 <init>\(\)V
```

Comment:
I'll amend your changes with adding `// 0 <init>\(LA;\)V` here. This is needed to check the `irClass.isInner` condition in `JvmDefaultConstructorLowering`, since the lowering is run before `InnerClassesLowering` where an additional parameter for the outer class is added.

Reply:
Thank you! I was sloppy in writing these tests and missed this part.